plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -111,7 +111,10 @@ export const PricingSection: React.FC = () => {
 							recordEvent( 'add_product_pricing_help' );
 						} }
 					>
-						How to price your product: expert tips
```

Comment:
Thank you for fixing this. I added this same fix on [this PR](https://github.com/woocommerce/woocommerce/pull/34769). I'll remove it later from mine.